### PR TITLE
Update style for failed actions

### PR DESCRIPTION
**Pull request by bduffany**
Completed actions with gRPC errors (e.g. INTERNAL) now render like this:

image

Completed actions with non-zero exit code now render like this:

image

When sorting by status (ascending) the order is now
- Started
- Queued
- Running
- Completed
  - Successful (gRPC OK, exitCode == 0)
  - RBE error (gRPC != OK, sorted by gRPC code)
  - Command error (gRPC == OK, exitCode != 0, sorted by exit code)

So to see failed actions first (which is our general sort order for other parts of the UI), the user just needs to sort by Status Desc, if it isn't sorted that way already.

---

**Version bump**: Patch 

**Related issues**: https://github.com/buildbuddy-io/buildbuddy-internal/issues/672

### CODE REVIEW

File: proto/defs.bzl [hunk 1]
```diff
@@ -71,7 +71,7 @@ def ts_proto_library(name, proto, **kwargs):
         # https://github.com/protobufjs/protobuf.js/tree/6.8.8#pbjs-for-javascript
         args = [
             "--target=static-module",
-            "--wrap=default",
```
Comment (bduffany) [inline proto/defs.bzl:74]: This change is unrelated -- it just outputs more readable generated TS proto code.

**Review by siggisim — APPROVED** [review]
